addons/sale_timesheet/models/product.py

```diff
@@ -76,7 +76,12 @@ def _compute_product_tooltip(self):
     @api.onchange('type', 'service_type', 'service_policy')
     def _onchange_service_fields(self):
         for record in self:
-            if record.type == 'service' and record.service_type == 'timesheet' and \
+            default_uom = self.env['ir.default']._get('product.template', 'uom_id',
+                                               company_id=self.env.company.id,
+                                               user_id=self.env.uid)
```

Comment:
In this case if the `ir.default` record doesn't have a user or company it will return none.
```suggestion
            default_uom = self.env['ir.default']._get_model_defaults('product.template').get('uom_id')
```
I think if we used `_get_model_defaults` it would be better as in the query fetching the records in this method we check if the user or the company is null or set by the env values which would be better I guess and will cover more cases. What do you think?